<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui</class>
 <widget class="QMainWindow" name="Gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="wynSym">
       <attribute name="title">
        <string>Wyniki symulacji</string>
       </attribute>
       <widget class="QCustomPlot" name="m_wykrWyjWartZad" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>621</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
       <widget class="QCustomPlot" name="m_wykrSter" native="true">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>270</y>
          <width>621</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>40</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Bieżąca wartość wyjścia</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>90</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Bieżąca wartość zadana</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelWyjscie">
        <property name="geometry">
         <rect>
          <x>649</x>
          <y>63</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>270</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Bieżąca sterowanie</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelWZadana">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>120</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelSterowanie">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>294</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QPushButton" name="symCiagla">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>150</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Symulacja ciągła</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="symKrokowa">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>180</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Symulacja krokowa</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="setWZadana">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>340</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="setDh">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>390</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>320</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Wartość zadana/ster</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>370</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Czas próbkowania</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="setPrSym">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>440</y>
          <width>53</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>420</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Prędkość symulacji</string>
        </property>
       </widget>
       <widget class="QPushButton" name="resetujSymulacje">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>210</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Resetuj symulację</string>
        </property>
       </widget>
       <widget class="QPushButton" name="resetujWykresy">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>240</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Resetuj wykresy</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="ustOb">
       <attribute name="title">
        <string>Ustawienia obiektu</string>
       </attribute>
       <widget class="QLineEdit" name="setWielomianLicznka">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>421</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="setWielomianMianownika">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>421</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>231</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Współczynniki wielomianu licznika</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>241</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Współczynniki wielomianu mianownika</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelWielomianLicznika">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>421</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="labelWielomianMianownika">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>421</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QSpinBox" name="wyborLicznika">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>10</y>
          <width>53</width>
          <height>24</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>2</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="wyborMianownika">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>100</y>
          <width>53</width>
          <height>24</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>2</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="setWariancja">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>200</y>
          <width>113</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>180</y>
          <width>57</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>Wariancja</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelWariancja">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>180</y>
          <width>57</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>833</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>&amp;Plik</string>
    </property>
    <addaction name="actionZakoncz"/>
    <addaction name="actionZapi_sz"/>
    <addaction name="actionWczytaj_plik"/>
   </widget>
   <widget class="QMenu" name="menuSymulacja">
    <property name="title">
     <string>&amp;Symulacja</string>
    </property>
    <addaction name="actionSymulacja_ciagla"/>
    <addaction name="actionSymulacja_krokowa"/>
    <addaction name="actionZatrzymaj_symulacje"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuSymulacja"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionZakoncz">
   <property name="text">
    <string>Zakończ</string>
   </property>
  </action>
  <action name="actionSymulacja_ciagla">
   <property name="text">
    <string>Symulacja ciągła</string>
   </property>
  </action>
  <action name="actionSymulacja_krokowa">
   <property name="text">
    <string>Symulacja krokowa</string>
   </property>
  </action>
  <action name="actionZatrzymaj_symulacje">
   <property name="text">
    <string>Zatrzymaj symulację</string>
   </property>
  </action>
  <action name="actionZapi_sz">
   <property name="text">
    <string>Zapi&amp;sz</string>
   </property>
  </action>
  <action name="actionWczytaj_plik">
   <property name="text">
    <string>Wczytaj plik</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
